src/test/java/org/aksw/iguana/cc/worker/impl/SPARQLProtocolWorkerTest.java

```diff
@@ -58,11 +69,12 @@ public class SPARQLProtocolWorkerTest {
     public static void setup() throws IOException {
         queryFile = Files.createTempFile("iguana-test-queries", ".tmp");
         Files.writeString(queryFile, QUERY, StandardCharsets.UTF_8);
-        SPARQLProtocolWorker.initHttpClient(1);
+        wm.setGlobalFixedDelay(5);
```

Comment:
What is the delay intended to trigger?

Reply:
One of the tests were failing because of wiremock. It seemed to lock itself due to high workload. (i think)
Limiting the rate at which request are processed by adding a fixed delay seemed to fix the problem.
The original problem only occured on the github runners.